lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
Thanks for the pointers! I've been using it wrong then.

Would it be safe to say that once this is in place, staging by default would be useful and recommended by Skylight?

Not sure how far along you are with it, but in case it helps: 

I personally enjoy the UX perspective of Airbrake, Honeybadger and Segment, as they handle separate environments quite well in-app as well as their installation process (which are mainly different keys or auto-parsing ENV information). From a security standpoint, different keys per environment are ideal (we often can give access to a developer to all-expect-production environments), though not a major gain (it's very rare we've used that).

It looks like you're still in very early product mode with Skylight -- just Intercom and email for support -- if you host a public [product feedback forum](https://digitalocean.uservoice.com/forums/136585-digitalocean), I'd be happy to contribute.